lib/rack/attack/store_proxy/redis_store_proxy.rb

```diff
@@ -4,6 +4,13 @@ module Rack
   class Attack
     module StoreProxy
       class RedisStoreProxy < SimpleDelegator
+        def initialize(*args)
```

Comment:
This is not being called because there's already an `#initialize` method a couple of lines below, which is overriding this one.